.gitignore

```diff
@@ -1,3 +1 @@
 node_modules
-package-lock.json
-yarn.lock
```

Comment:
Only one package manager should be used in the project, so no need to exclude both files. Or we should make sure (somehow) that used only one designated PM.

Reply:
I found many times that deleting package-lock.json (or yarn. lock) may break the app on the other machine. Better to avoid that and always commit changes of *.lock file.
if you use another package manager, so it means you are going "delete-ignore" the previous. And it will lead to an inconsistent dependency tree.

Reply:
I agree, we should always use npm for this package. Removing both lockfiles from the gitignore file will help us do that because accidental use of yarn will result in a different lockfile being created (which should be a signal in the diff).